arch/setup.sh

```diff
@@ -6,7 +6,7 @@ makepkg -si
 cd ..
 rm -rf yay
 
-sudo yay -Sy fortune-mod figlet zsh htop git ncdu nano base-devel wget curl aria2 inetutils bat ripgrep p7zip ookla-speedtest-bin
```

Comment:
why get rid of git and base-devel?

Reply:
git would be needed for cloning repo and base-devel is a requirement for building yay before

Reply:
I could probably add "minimum requirement: a user with sudo privileges and git and base-devel installed" somewhere to clarify maybe?

Reply:
https://github.com/gotenksIN/scripts/commit/6cb501a5fc37765a6773181e61c3b8131f5fcc8d should fix it